aws-observability-terraform/main_variables.auto.tfvars

```diff
@@ -0,0 +1,56 @@
+# Sumo Logic - AWS Observability Solution Terraform
+
+# This terraform variables file configures credentials and parameters for AWS Observability CloudFormation stack.
+# For more information on Setting up AWS Observability, Please visit - https://help.sumologic.com/Solutions/AWS_Observability_Solution/03_Set_Up_the_AWS_Observability_Solution
+
+## AWS CloudFormation Stack Name. Provide a unique AWS CloudFormation Stack Name.
+# Please replace <YOUR STACK NAME> (including brackets) with stack name.
+CloudFormationStackName = "<YOUR STACK NAME>"
+
+# For more details on parameters, Please visit - https://help.sumologic.com/Solutions/AWS_Observability_Solution/03_Set_Up_the_AWS_Observability_Solution#Configuration_prompts_and_input
+
+####### BELOW ARE REQUIRED PARAMETERS FOR CLOUDFORMATION STACK #######
+# Visit - https://help.sumologic.com/Solutions/AWS_Observability_Solution/03_Set_Up_the_AWS_Observability_Solution#sumo-logic-access-configuration-required
+Section1aSumoLogicDeployment = "<YOUR SUMO DEPLOYMENT>" # Please replace <YOUR SUMO DEPLOYMENT> (including brackets) with au, ca, de, eu, jp, us2, in, fed or us1.
```

Comment:
Why the variables are named as SectionXVariableName. Is it helpful to customers in anyway?

Reply:
This is done in order to keep sync between Cloudformation template parameters and terraform parameters.